eclass/qt5-build.eclass

```diff
@@ -213,6 +213,11 @@ qt5-build_src_prepare() {
 		# Don't add -O3 to CXXFLAGS (bug 549140)
 		sed -i -e '/CONFIG\s*+=/ s/optimize_full//' \
 			src/{corelib/corelib,gui/gui}.pro || die "sed failed (optimize_full)"
+
+		# Respect build variables in configure tests (bug #639494)
+		if [[ ${QT5_MINOR_VERSION} -ge 9 ]]; then
+			sed -i -e "s|\"\$outpath/bin/qmake\" \"\$relpathMangled\" -- \"\$@\"|& $(qt5_qmake_args) |" configure || die
```

Comment:
I think we could achieve the same thing by passing those args to ``configure`` in ``qt5_base_configure`` but I can't get the quoting to work correctly.